src/core/dswp/src/DSWP.cpp

```diff
@@ -52,6 +56,13 @@ bool DSWP::canBeAppliedToLoop(LoopContent *LDI, Heuristics *h) const {
     return false;
   }
 
+  /*
+   * Exclude OutputSequenceSCC, which does not support DSWP
+   */
+  auto SCCManager = LDI->getSCCManager();
+  auto hasAnyOutputSequenceSCC =
```

Comment:
Why is this computed in DSWP?

Reply:
We wrote this in person last week, I initially had it as a method of OutputSequenceSCC but that broke separation of concerns I think.

Reply:
Per IRL: Oops put the return back